spec/graphql/tracing/data_dog_tracing_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe GraphQL::Tracing::DataDogTracing do
+  module DataDogTest
+    class Query < GraphQL::Schema::Object
+      add_field GraphQL::Types::Relay::NodeField
```

Comment:
```suggestion
  include GraphQL::Types::Relay::HasNodeField
```

There was a bug here fixed in master since this branch came off: #3791

Reply:
I'll merge this and fix it on master